tests/js/session-tests.js

```diff
@@ -367,8 +384,6 @@ module.exports = {
         };
         const realm = new Realm(config);
         const session = realm.syncSession;
-
-        TestCase.assertEqual(session.config.onError, config.sync.onError);
```

Comment:
This assert is removed in master as well, not sure if I should do it here also

Reply:
It should be removed